include/librealsense2/rsutil.h

```diff
@@ -96,6 +96,15 @@ static void rs2_deproject_pixel_to_point(float point[3], const struct rs2_intrin
         x = ux;
         y = uy;
     }
+    if (intrin->model == RS2_DISTORTION_BROWN_CONRADY)
```

Comment:
Didn't you also say there was a bug in the brown-conrady projection code?

Reply:
done